lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -829,17 +899,29 @@ def remove_toolitem(self, name):
                     self._groups[group].remove(toolitem)
         del self._toolitems[name]
 
+    def set_flow(self, flow):
```

Comment:
Yes at insertion we create the toolbar. That is why I think the best is a
ToolbarBase kwarg
On 27 Jun 2015 11:19 am, "OceanWolf" <notifications@github.com> wrote:

> In lib/matplotlib/backends/backend_gtk3.py
> <https://github.com/matplotlib/matplotlib/pull/4143#discussion_r33413283>:
>
> > @@ -829,17 +899,29 @@ def remove_toolitem(self, name):
> >                      self._groups[group].remove(toolitem)
> >          del self._toolitems[name]
> >
> > +    def set_flow(self, flow):
>
> Actually I think this could combine quite nicely with your idea to add
> replace/remove element. When inserting an element we could set the flow on
> the toolbar, i.e. vertical for the east/west and horizontal for north/south.
>
> Maybe we should just indicate that we don't expect the user to change this
> manually?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/4143/files#r33413283>.
>